test/expect-tests/dune_pkg/dune

```diff
@@ -29,7 +29,7 @@
  (target tarball.tar.gz)
  (deps plaintext.md)
  (action
-  (run tar -czf %{target} %{deps})))
+  (run tar -czf %{target} -C .. dune_pkg/%{deps})))
```

Comment:
I would've probably moved `plaintext.md` into a subfolder to avoid `tar` escaping out of the folder, but this also works.

Reply:
Good point. I moved the inputs to a subdir and also added another file into the archive just for thoroughness.